saltdocker.py

```diff
@@ -42,12 +42,12 @@ def __init__(self, version):
             
             if latest is True:
                 args.extend([
-                    '--tag', 'gtmanfred/saltstack:latest',
+                    '--tag', 'saltstack/salt-master:latest',
```

Comment:
you don't want this to be just the salt master it can also run the minion, and you could potentailly want to add proxy minions.